lib/components/timer_component.dart

```diff
@@ -1,4 +1,5 @@
 import 'dart:ui';
+
 import './component.dart';
```

Comment:
Should we have a standard for whether they should have `./` maybe?
Because now there is a mix.

Reply:
Good point, I will fix my plugin to unify that and fix in a followup.

Reply:
Fixed here https://github.com/flame-engine/flame/pull/501